src/ui/common/src/components/workflows/workflowHeader.tsx

```diff
@@ -43,8 +51,12 @@ const WorkflowHeader: React.FC<Props> = ({ user, workflowDag }) => {
   const [showSuccessToast, setShowSuccessToast] = useState(false);
   const [showSettings, setShowSettings] = useState(false);
 
-  const handleSuccessToastClose = () => {
+  const handleSuccessToastClose = async () => {
```

Comment:
Here's the main code that goes to the newly created workflow run after the user selects run workflow on the UI.